gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -239,11 +246,26 @@ private GoogleCloudStorageItemInfo fetchInitialMetadata() throws IOException {
               RetryDeterminer.SOCKET_ERRORS,
               IOException.class,
               sleeper);
+
+      logger.atFinest().log(
+          "GoogleCloudStorageReadChannel:getMetadata complete context:%d,time:%d,resource:%s,requestId:%s",
```

Comment:
we don't need to log the class name `GoogleCloudStorageReadChannel`  logging pattern have the fully qualified class name